sdk/communication/Azure.Communication.Sms/tests.yml

```diff
@@ -11,11 +11,13 @@ extends:
           - $(sub-config-azure-cloud-test-resources)
           - $(sub-config-communication-services-cloud-test-resources-common)
           - $(sub-config-communication-services-cloud-test-resources-net)
+        MatrixReplace:
+          - CollectCoverage=.*/true
```

Comment:
While this should be true by default, from what I can tell in this PR, this PR disables it for all CIs which is a regression.

Reply:
This PR retains the current behavior, which is opt-out for CI and opt-in for live:

Here's where I turn it on for all CI pipelines:
https://github.com/Azure/azure-sdk-for-net/pull/21277/files#diff-8846d7741afb575539571d7b46758247229daaa4336751c6b3e743fd284426faR85